sesman/sesman.c

```diff
@@ -417,6 +417,17 @@ main(int argc, char **argv)
     log_message(LOG_LEVEL_INFO,
                 "starting xrdp-sesman with pid %d", g_pid);
 
+    /* make sure socket (XRDP_SOCKET_PATH) directory exists */
```

Comment:
Where does that go?  I think debian likes to put it in the socket path.   It needs to be deterministic because sound needs it.